frontend/survey/surveyDirective.js

```diff
@@ -106,9 +106,24 @@
             return surveyCtrl.post.title && !notEnoughOptions && surveyCtrl.post.deadline;
         };
 
+        function defaultoptions(){
+            surveyCtrl.options = [];
+            surveyCtrl.options.push(angular.copy(option_empty));
```

Comment:
All this function could just be this:
`surveyCtrl.options = [];`
`_.range(MIN_QUANTITY_OPTION).map(() => surveyCtrl.options.push(angular.copy(option_empty)))`

or
```
function reducer (acc) {
   acc.push(angular.copy(option_empty))
   return acc
}
survey.options = _.range(MIN_QUANTITY_OPTION).reduce(reducer, [])

Reply:
or just a `for` iterating over MIN_QUANTITY_OPTION